public/src/components/promoTool/promoEditor.tsx

```diff
@@ -375,6 +402,27 @@ const PromoEditor = ({
         </Box>
       </div>
 
+      <div className={classes.section}>
+        <Typography className={classes.sectionTitle}>Landing page</Typography>
+        <FormControlLabel
+          control={
+            <Checkbox
+              checked={promotionHasLandingPage}
+              onChange={handlePromotionHasLandingPageChange}
+              disabled={!isEditing}
+            />
+          }
+          label="This promotion has a landing page"
+        />
+        {promotionHasLandingPage && (
+          <PromoLandingPage
```

Comment:
we only want to show this component if the product is newspaper or guardian weekly